Missing header unistd.h in emit_cpp.c

```
  CC     emit_cpp.o
emit_cpp.c: In function ‘emit_header_start’:
emit_cpp.c:306:19: warning: implicit declaration of function ‘_exit’ [-Wimplicit-function-declaration]
                   _exit(1);
                   ^
emit_cpp.c:306:19: warning: incompatible implicit declaration of built-in function ‘_exit’ [enabled by default]
```

--- a/lcmgen/emit_cpp.c
+++ b/lcmgen/emit_cpp.c
@@ -6,6 +6,8 @@
 #include <ctype.h>
 #ifdef WIN32
 #define __STDC_FORMAT_MACROS			// Enable integer types
+#else
+#include <unistd.h> /* _exit */
 #endif
 #include <inttypes.h>
 

